templates/zerver/help/include/rest-endpoints.md

```diff
@@ -1,65 +1,65 @@
-* [Get topics in a stream](/api/get-stream-topics)
-* [Topic muting](/api/mute-topics)
+* [Get all streams](/api/get_streams)
+* [Get subscribed streams](/api/list_subscriptions)
+* [Add subscriptions](/api/subscribe)
+* [Update subscription settings](/api/update_subscription_settings)
+* [Remove subscriptions](/api/unsubscribe)
+* [Get topics in a stream](/api/get_stream_topics)
+* [Topic muting](/api/mute_topic)
 * [Create a stream](/api/create-stream)
-* [Get stream ID](/api/get-stream-id)
-* [Delete a stream](/api/delete-stream)
-* [Update a stream](/api/update-stream)
+* [Get stream ID](/api/get_stream_id)
+* [Delete a stream](/api/delete_stream)
+* [Update a stream](/api/update_stream)
 
 #### Users
 
-* [Get all users](/api/get-all-users)
-* [Get a user](/api/get-user)
-* [Deactivate a user](/api/deactivate-user)
-* [Reactivate a user](/api/reactivate-user)
-* [Update a user](/api/update-user)
-* [Get profile](/api/get-profile)
-* [Create a user](/api/create-user)
-* [Set "typing" status](/api/typing)
-* [Get user presence](/api/get-presence)
-* [Get all user groups](/api/get-user-groups)
-* [Update notification settings](/api/update-notification-settings)
-* [Create a user group](/api/create-user-group)
-* [Update a user group](/api/update-user-group)
-* [Delete a user group](/api/delete-user-group)
+* [Get all users](/api/get_users)
+* [Get a user](/api/get_user_by_id)
+* [Deactivate a user](/api/deactivate_user_by_id)
+* [Reactivate a user](/api/reactivate_user)
+* [Update a user](/api/update_user)
+* [Get profile](/api/get_profile)
+* [Create a user](/api/create_user)
+* [Set "set_typing_status" status](/api/set_typing_status)
+* [Get user presence](/api/get_user_presence)
+* [Get all user groups](/api/get_user_groups)
+* [Update notification settings](/api/update_notification_settings)
+* [Create a user group](/api/create_user_group)
+* [Update a user group](/api/update_user_group)
+* [Delete a user group](/api/remove_user_group)
 
 #### Server & organizations
 
-* [Get server settings](/api/server-settings)
-* [List linkifiers](/api/list-linkifiers)
-* [Create linkifiers](/api/add-linkifiers)
-* [Remove linkifiers](/api/remove-linkifiers)
-* [Get all custom emoji](/api/get-org-emoji)
-* [Upload custom emoji](/api/upload-custom-emoji)
+* [Get server settings](/api/get_server_settings)
+* [List linkifiers](/api/get_realm_filters)
+* [Create linkifiers](/api/add_realm_filter)
+* [Remove linkifiers](/api/remove_realm_filter)
+* [Get all custom emoji](/api/get_realm_emoji)
+* [Upload custom emoji](/api/upload_custom_emoji)
 
 #### Real-time events
 
 * [Real time events API](/api/real-time-events)
-* [Register an event queue](/api/register-queue)
-* [Get events from an event queue](/api/get-events-from-queue)
-* [Delete an event queue](/api/delete-queue)
+* [Register an event queue](/api/register_queue)
+* [Get events from an event queue](/api/get_events)
+* [Delete an event queue](/api/deregister)
```

Comment:
"deregister" sound like potentially about one's account, the same way plain "register" does. How about `delete_queue`? That seems nicely parallel to `register_queue`.

Reply:
Yeah, agreed, `delete_queue` is much clearer.